lib/coverband/adapters/base.rb

```diff
@@ -124,8 +130,15 @@ def merge_expanded_data(new_expanded, old_expanded)
         }
       end
 
+      # TODO: This should only be 2 cases get our dup / not dups aligned
       def array_add(latest, original)
```

Comment:
throughout our code we now have code paths that either have a froze or not frozen coverage... this creates problems... in the end we probably want to pick ONE way of going starting with a full set and using in memory `!` or starting with frozen and then always reducing to new....